userdata/rundsc.ps1

```diff
@@ -367,6 +367,17 @@ function New-LocalCache {
   }
 }
 
+# SourceRepo is in place to toggle between production and testing environments
+#$evn:SourceRepo = mozilla-releng
+$SourceRepo = "markcor"
```

Comment:
Not currently. I added just so there was a quick manual switch when someone has forked the repo. We could do an auto-detection for AWS instances, but for hardware it would be tricky. Since we are not passing any type of metadata to the hardware nodes.